### PR TITLE
SL-192 ED triage changes for SL.  More to come...

### PR DESCRIPTION
**Pull request by lnball**
Still working on this branch and need more information from the KGH ED team this week.  But I wanted you to see the types of changes on this branch for SL.  These would not work for ZL.  Added English translations for things that are not translated to French.

### CODE REVIEW

**Review by cioan — APPROVED** [review]
LGTM! Thanks @lnball !

**Comment by mogoodrich** [comment]
Thanks @lnball .  My gut right now is that a global property specifying the Haiti or SL  configuration may be the way to go.  We will have to do detailed testing on both Haiti and SL to make sure we have SL working as expected and that there aren't any regressoins for Haiti.

**Comment by mogoodrich** [comment]
Also converted this to a draft while it's a work-in-progress.

**Comment by lnball** [comment]
Draft?  Is that a GitHub thing so it’s not accidentally merged into master?

Sent from my iPhone

On Mar 1, 2023, at 2:29 PM, Mark Goodrich ***@***.***> wrote:

﻿

Also converted this to a draft while it's a work-in-progress.

**Comment by mogoodrich** [comment]
Yes, just a github way to flag that the PR is a work-in-progress...

File: api/src/main/resources/messages.properties [hunk 1]
```diff
@@ -80,34 +83,63 @@ edtriageapp.formHistory=Form History
 
 # symptoms that need translations
 edtriageapp.117617AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=Pregnancy & trauma or vaginal bleeding
-edtriageapp.120977AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=Burn - facial/inhalation
```
Comment (mogoodrich) [inline api/src/main/resources/messages.properties:83]: Just to confirm @lnball  we want to maintain all these English translations in Haiti, or there any we want toc change everywhere?  Are this all the "official" South African triage English names and so we want to keep a copy of them for that reason?

**Comment by lnball** [comment]
Haiti names should remain unchanged.  The users in SL require descriptive names because they are not always doctors or nurses.

I can explain better tomorrow, Mark.

On Mar 8, 2023, at 2:12 PM, Mark Goodrich ***@***.***> wrote:

﻿

@mogoodrich commented on this pull request.

________________________________

In api/src/main/resources/messages.properties<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FPIH%2Fopenmrs-module-edtriageapp%2Fpull%2F5%23discussion_r1129916685&data=05%7C01%7Ceball%40pih.org%7C33a8a596b6e84949fe0e08db200908d1%7C5254789f6860437585bc302509fad508%7C0%7C0%7C638138995401568167%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=w4UGki5le0tZcFd6%2FwimNnxTcnO9JRt%2BM8jpQ95TP2o%3D&reserved=0>:

> @@ -80,34 +83,63 @@ edtriageapp.formHistory=Form History

 # symptoms that need translations
 edtriageapp.117617AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=Pregnancy & trauma or vaginal bleeding
-edtriageapp.120977AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=Burn - facial/inhalation


Just to confirm @lnball<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Flnball&data=05%7C01%7Ceball%40pih.org%7C33a8a596b6e84949fe0e08db200908d1%7C5254789f6860437585bc302509fad508%7C0%7C0%7C638138995401568167%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=BEunRxaLNO%2B4b0urrGWT%2FSmawu0GiqIn2WSq392bo40%3D&reserved=0> we want to maintain all these English translations in Haiti, or there any we want toc change everywhere? Are this all the "official" South African triage English names and so we want to keep a copy of them for that reason?

**Comment by lnball** [comment]
@mogoodrich Did you want to discuss?  There are a couple things that I think will work for both ZL and SL.  

- changed "Pulse (english)" to "Heart failure (english)".  It might be different on the SATS form, but the meaning is the same.
- The translations.gsp file is a crazy way of listing every concept which might would use the message property instead of the preferred name.  Maybe it will work unchanged and show the preferred name if the concept isn't in the message property file?

**Comment by mogoodrich** [comment]
Yeah, we can discuss, the main thing was any changes that you made that we want to keep for Haiti as well.

And I do believe that if a concept is not in the translations file, it just uses the preferred name... are there some of the translations you added that you don't think we need if it works if it uses the preferred name?

**Comment by mogoodrich** [comment]
Actually, you are right, it might not be displaying the preferred name by default...

**Pull request closed without merging**